cmd/callVariants/callVariants.go

```diff
@@ -27,15 +27,15 @@ func usage() {
 	flag.PrintDefaults()
 }
 
-func callVariants(experimentalFiles, normalFiles []string, refFile, outFile string, maxP, minAf float64, minCoverage, workers int) {
+func callVariants(experimentalFiles, normalFiles []string, refFile, outFile string, maxP, minAf, maxAf, maxStrandBias float64, minCoverage, minMapQ, minAltReads, workers int) {
```

Comment:
perhaps enough options for a Settings struct, especially if this cmd is likely to grow in complexity

Reply:
I thought about it, but they are being used in various different places downstream, so I opted to not use a struct.